<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>EditPersonalDataDlgDecl</class>
  <widget class="QDialog" name="EditPersonalDataDlgDecl">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>546</width>
        <height>448</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>New File Dialog</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>6</number>
      </property>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="TextLabel1_2">
              <property name="text">
                <string>Use this dialog to enter personal information about yourself.

All information is optional and is provided to personalize
your KMyMoney file.</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Line" name="Line1">
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
            <number>1</number>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QGridLayout">
          <item row="3" column="0">
            <widget class="QLabel" name="TextLabel4">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string>County/State:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="KLineEdit" name="userNameEdit"/>
          </item>
          <item row="3" column="1">
            <widget class="KLineEdit" name="countyEdit"/>
          </item>
          <item row="2" column="1">
            <widget class="KLineEdit" name="townEdit"/>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="TextLabel2">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string>Street:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="KLineEdit" name="streetEdit"/>
          </item>
          <item row="6" column="0">
            <widget class="QLabel" name="TextLabel7">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string comment="Email address">Email:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="0">
            <widget class="QLabel" name="TextLabel1">
              <property name="text">
                <string>Your name:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="5" column="0">
            <widget class="QLabel" name="TextLabel6">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string>Telephone:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QLabel" name="TextLabel3">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string>Town:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="6" column="1">
            <widget class="KLineEdit" name="emailEdit"/>
          </item>
          <item row="4" column="0">
            <widget class="QLabel" name="TextLabel5">
              <property name="minimumSize">
                <size>
                  <width>100</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string>Postal Code:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="5" column="1">
            <widget class="KLineEdit" name="telephoneEdit"/>
          </item>
          <item row="4" column="1">
            <widget class="KLineEdit" name="postcodeEdit"/>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <widget class="QPushButton" name="kabcBtn">
              <property name="text">
                <string>Load from Addressbook</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer11">
              <property name="sizeHint">
                <size>
                  <width>305</width>
                  <height>21</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <spacer name="spacer2">
          <property name="sizeHint">
            <size>
              <width>20</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer name="Spacer1">
              <property name="sizeHint">
                <size>
                  <width>360</width>
                  <height>16</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="okBtn">
              <property name="text">
                <string>OK</string>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelBtn">
              <property name="text">
                <string>Cancel</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
    <tabstops>
    <tabstop>userNameEdit</tabstop>
    <tabstop>streetEdit</tabstop>
    <tabstop>townEdit</tabstop>
    <tabstop>countyEdit</tabstop>
    <tabstop>postcodeEdit</tabstop>
    <tabstop>telephoneEdit</tabstop>
    <tabstop>emailEdit</tabstop>
    <tabstop>kabcBtn</tabstop>
    <tabstop>okBtn</tabstop>
    <tabstop>cancelBtn</tabstop>
  </tabstops>
  <customwidgets>
    <customwidget>
      <class>KLineEdit</class>
      <extends>QLineEdit</extends>
      <header location="global">klineedit.h</header>
    </customwidget>
  </customwidgets>
</ui>
